llvm/test/CodeGen/RISCV/rvv/fixed-vectors-shuffle-reverse.ll

```diff
@@ -874,27 +874,30 @@ define <16 x i8> @reverse_v16i8_2(<8 x i8> %a, <8 x i8> %b) {
 define <32 x i8> @reverse_v32i8_2(<16 x i8> %a, <16 x i8> %b) {
```

Comment:
For context, all of the reverse test diffs are because we get a mask of the form: <3,2,1,0, undef, undef, undef, undef>

This is being recognized as a locally repeating mask (this change) which is perfectly valid.  However, we could better here by exploiting the fact that only one vreg is actually being manipulated here.  I'll explore that in a following change.

Reply:
Here's the mentioned change: https://github.com/llvm/llvm-project/pull/125768